CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/compare/CompareGraphPlugin.java

```diff
@@ -484,6 +480,7 @@ protected GraphRecordStore compareGraphs(final String title, final GraphRecordSt
 
                         switch (keyType) {
                             case "source":
+                                break;
```

Comment:
similar

Reply:
I added the explicit behaviour because in my opinion, the readability increases when explicitly mentioning that the case does nothing and will break. I've changed this back in the most recent commit.